components/Home.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
```

Comment:
Link isn't currently being used here - don't forget to remove any unused imports.